src/Microsoft.DotNet.Build.Tasks/PackageFiles/Build.Common.props

```diff
@@ -41,7 +41,7 @@
     <BuildToolsTaskDir Condition="'$(RunningOnCore)'!='true'">$(ToolsDir)net46/</BuildToolsTaskDir>
     <BuildToolsTaskDir Condition="'$(RunningOnCore)'=='true'">$(ToolsDir)</BuildToolsTaskDir>
     <UseRoslynCompilers Condition="'$(UseRoslynCompilers)'=='' and '$(RunningOnUnix)'=='true'">false</UseRoslynCompilers>
-    <UseSharedCompilation Condition="'$(UseSharedCompilation)' == '' and '$(RunningOnUnix)' != 'true' and '$(UseRoslynCompilers)' != 'false'">true</UseSharedCompilation>
+    <UseSharedCompilation>false</UseSharedCompilation>
```

Comment:
That's really going to hurt build throughput. Why is this happening?

Reply:
yeah, I've seen builds are much slower than before, but this is required since this is a very recent build of roslyn which doesn't support msbuild 14.0 anymore. since we don't want to drop support from that version of msbuild just yet, we have to add this workaround and manually set csc path or else we would get an error like: `Microsoft.Net.Compilers is only supported on MSBuild v15.0 and above`

Reply:
Would you suggest to have something like this instead:
```xml
<UseSharedCompilation Condition="'$(UseSharedCompilation)' == '' and '$(RunningOnUnix)' != 'true' and '$(UseRoslynCompilers)' != 'false' and '$(VisualStudioVersion)' != '14.0'">true</UseSharedCompilation>
```

Where I check the VisualStudioVersion to see if its 14.0 or not?

Reply:
> since we don't want to drop support from that version of msbuild just yet

When do we plan on dropping support for it?  VS 2017 has been out for 6 months.

Reply:
just chatted with @weshaggard about that, the problem is that these changes are in buildtools and we don't want to assume that it's ok to drop support of VS2015 on all of the repos that consume buildtools. Being a general repo that gets consumed by a lot of repos, we rather keep these changes compatible with msbuild 14.0 for the moment.

Reply:
Thought the point of BuildTools was a light weight way to provide tooling to repos. But it can't support the latest C# compiler? That seems bad.

Reply:
That's why I just changed the condition, what we want is to support all versions of C# compiler and let the repo decide which one to use, as opposed to only supporting the newer version or the older version.